configs/opencv-ovms/grpc_go/main.go

```diff
@@ -123,19 +108,13 @@ func runModelServer(client *grpc_client.GRPCInferenceServiceClient, webcam *gocv
 		if err != nil {
 			fmt.Printf("post process failed: %v\n", err)
 		}
-		fmt.Printf("length of detectedObjects after  processing: %v\n", len(detectedObjects.Objects))
-		detectedObjects = detectedObjects.FinalPostProcessAdvanced()
```

Comment:
I believe we need this line back

Reply:
yea we need `detectedObjects = detectedObjects.FinalPostProcessAdvanced()` line back, that is a good catch!